helix-core/src/main/java/org/apache/helix/controller/GenericHelixController.java

```diff
@@ -1165,7 +1163,11 @@ public void run() {
               + _processorName);
       while (!isInterrupted()) {
         try {
-          handleEvent(_eventBlockingQueue.take(), _cache);
+          ClusterEvent newClusterEvent = _eventBlockingQueue.take();
+          String threadName = String.format(
+              "HelixController-pipeline-%s-(%s)", _processorName, newClusterEvent.getEventId());
```

Comment:
can we include clusterName here in the thread name? What is the processor name here?

Reply:
`processorName` is actually the cluster name, I don't want to touch the other places unless necessary. The name convention is aligned with 
`super("HelixController-pipeline-" + processorName);` which sets the ThreadName in the constructor.